boards/shields/nrf21540_ek/doc/index.rst

```diff
@@ -0,0 +1,71 @@
+.. _nrf21540_ek:
+
+nRF21540 evaluation kit
+#######################
+
+The nRF21540 EK is an RF front-end module (FEM) for Bluetooth Low Energy, Bluetooth mesh, 2.4 GHz proprietary, Thread, and Zigbee range extension.
```

Comment:
Please wrap lines at 78 or 80 characters to help people who will be looking at this as a text file.

Reply:
Done